tip-wlan/charts/wlan-prov-service/resources/config/datasource.properties

```diff
@@ -1,11 +1,11 @@
 singleDataSource.url=jdbc:postgresql://tip-wlan-postgresql:5432/prov_db
-singleDataSource.username=DUMMY_POSTGRES_TIP_USER
-singleDataSource.password=DUMMY_POSTGRES_TIP_PASSWORD
+singleDataSource.username=tip_user
+singleDataSource.password=tip_password
```

Comment:
Do you plan to commit the passwords in the charts?

A better approach would be to convert such files into configmaps, get the values of these parameters from values.yaml and then override these from dev-*.yaml file.

Reply:
they are already hardcoded everywhere .. refactoring this may make another good PR @rahulsharma2020

Reply:
@rahulsharma2020 does this look ok to you? https://github.com/Telecominfraproject/wlan-cloud-helm/pull/9/files#diff-d606bccc7c86fa7a06b288c3d53be757668c4d61b649419ee337fdfceeac7b4cR8